app/assets/javascripts/components/story/StoryAttachment.js

```diff
@@ -58,7 +62,43 @@ class StoryAttachment extends React.Component {
     };
   }
 
+  componentWillMount() {
+    this.setAttachinaryOptions();
+  }
+
+  setAttachinaryOptions() {
+    this.getUpdatedSignature()
+      .then((response) => {
+        this.setState({
+          attachinaryOptions: response,
+        })
+      })
+  }
+
+  updateSignature() {
+    latestTimestamp = this.state.attachinaryOptions.html.data.form_data.timestamp * 1000;
```

Comment:
When creating local variables in JS you need to put a specifier of the kind (`let` or `const`) of the variable before its name or you'll create a global accidentally. Since you don't need to reassign the value of these variables, what do you think about using `const` here (you'd be using `let` if you needed to reassign it).

Something like:

```js
const latestTimestamp = this.state.attachinaryOptions.html.data.form_data.timestamp * 1000;
const currentTimestamp = new Date().getTime();
const elapsedTime = currentTimestamp - latestTimestamp;
const timeToInvalidate = 3600000;
```